book/src/external-resizer.md

```diff
@@ -8,7 +8,8 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-resizer v0.5.0](https://github.com/kubernetes-csi/external-resizer/tree/v0.5.0)  | [release-0.5](https://github.com/kubernetes-csi/external-resizer/tree/release-0.5) |[v1.1.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | - | quay.io/k8scsi/csi-resizer:v0.5.0 | v1.15 | - | v1.16
+[external-resizer v1.0.0](https://github.com/kubernetes-csi/external-resizer/tree/v1.0.0)  | [release-1.0](https://github.com/kubernetes-csi/external-resizer/tree/release-1.0) |[v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | - | k8s.gcr.io/sig-storage/csi-resizer:v1.0.0 | v1.15 | - | v1.16
```

Comment:
@gnufied csi-resizer release page says min v1.16. Is that correct?

Reply:
Yes I think 1.16 as minimum version makes sense. That is when CSI volume expansion went beta.

Reply:
updated